app/script/view_model/content/MessageListViewModel.js

```diff
@@ -414,33 +414,12 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
       .catch(() => reset_progress());
   }
 
-  /**
-   * Gets CSS class that will be applied to the message div in order to style.
-   * @param {z.entity.Message} message - Message entity for generating css class
-   * @returns {string} CSS class that is applied to the element
-   */
-  get_css_class(message) {
```

Comment:
In case you are wondering, this is safe to remove because this css classe was overridden anyway by the next `css` binding here https://github.com/wireapp/wire-webapp/pull/3745/files#diff-b0ae56dc64faa60de5486cf73f73ff0cL8